src/zoid/buttons/util.js

```diff
@@ -18,6 +18,38 @@ export function determineFlow(props : ButtonProps) : $Values<typeof BUTTON_FLOW>
     }
 }
 
+export function supportsQRPay(funding : $Values<typeof FUNDING>) : boolean {
+    if (funding === FUNDING.VENMO && !isIos() && !isAndroid()) {
```

Comment:
isDevice() may be a better check, just to make sure that no other mobile OS's leak in here -- https://github.com/krakenjs/belter/blob/8cf35443d53cb81fcb7bce0bdb385aeaffc7423c/src/device.js#L8

Reply:
Good point Mark. I was wondering about the requirements for when to show the venmo button. Plus one for using `!isDevice()` for determining eligibility for showing venmo for the QR Pay experience.

Reply:
yeah i use `isDevice()` later 🤔 

idk why I did that like that here...